src/screens/SelectionScreen/SelectionScreen.jsx

```diff
@@ -30,19 +23,23 @@ const SelectionScreen = () => {
     <UnauthenticatedLayout>
       <div className="select-srn-wrapper">
         <div className="select-srn-header-wrapper">
-          <h1 className="select-srn-title">LOG IN</h1>
-          <h1 className="select-srn-heading">SELECT YOUR OPTION</h1>
+          <h1 className="select-srn-title">{t("screens.selectionScreen.title").toUpperCase()}</h1>
```

Comment:
I've used css and text-transform for uppercase transformation. We should do it consistent. Should we use js oder css for that?

Reply:
Yes, we can use css for that.